src/pages/TCY/components/Unstake/UnstakeInput.tsx

```diff
@@ -206,14 +217,14 @@ export const UnstakeInput: React.FC<TCYRouteProps & { activeAccountNumber: numbe
       {headerComponent}
       <FormControl isInvalid={Boolean(errors.amountCryptoPrecision)}>
         <TradeAssetInput
-          accountId={accountId}
```

Comment:
We are now showing the staking balance for the user balance by removing this and replacing with `balance`. This seems incorrect

Reply:
I'm confused, isn't that's precisely what we want? 
<img width="1413" height="414" alt="image" src="https://github.com/user-attachments/assets/e550051e-67aa-48e0-b167-88f6907d779e" />